CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 0.1.11
+Released on January 23, 2023
+
+### Enhancements
+* Uses the elkjs library to improve node and edge positioning on the UI.
+* Shows the name of the options on the UI's menu sidesheet to improve clarity.
+* Removes the Aqueduct logo on the UI's home page to reduce redundancy.
+
+### Bugfixes
+* Fixes bug where operator execution fails when running on Kubernetes.
```

Comment:
These bugfixes could be a little clearer. How did the execution fail? Did it always happen or only in certain circumstances?